src/utils/aeternity.js

```diff
@@ -33,10 +29,10 @@ export const initClient = async () => {
   try {
     if (window.Cypress) {
       client = await Universal({
-        compilerUrl,
+        compilerUrl: COMPILER_URL,
         nodes: [{
           name: 'testnet',
-          instance: await Node({ url: nodeUrlTestNet, internalUrl: nodeUrlTestNet }),
+          instance: await Node({ url: 'https://testnet.aeternity.io' }),
```

Comment:
Can this also be extracted?

Reply:
why would we want to extract a constant for testing?

Reply:
Isn't this also used for mainnet?